features/step_definitions/basic_steps.rb

```diff
@@ -20,10 +20,9 @@
   end
 end
 
-Given(/^We have the following (?:recipes|list):$/) do |table|
+Given('We have the following recipes:') do |table|
   table.hashes.each do |recipe|
+    recipe["user_id"] = User.last.id
```

Comment:
Please use a symbol for the key. Do you really want the same user to be assigned to the recipe like this? If yes, let the Factory do its job